scripts/instrument-salesforcedx-vscode-extensions.js

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env node
+
+const shell = require('shelljs');
+shell.set('-e');
+shell.set('+v');
+
+const path = require('path');
+
+const extensionsToInstrument = [
```

Comment:
If we add a new module this will need to be updated.
I don't want to just scan the modules under /packages automatically since they could be packages that we don't care about in terms of code coverage.